src/RecoveryServices/RecoveryServices.SiteRecovery/NetworkInterfaceCard/NewAzureRmAsrVmNicConfig.cs

```diff
@@ -253,6 +253,28 @@ public override void ExecuteSiteRecoveryCmdlet()
                             vmNic.EnableAcceleratedNetworkingOnTfo ?? false;
                     }
 
+                    List<PSIPConfigInputDetails> ipConfigList = null;
+                    if (this.IPConfig == null || this.IPConfig.ToList().Count == 0)
+                    {
+                        ipConfigList = vmNic.IpConfigs?.Select(ip => ConvertToPSIPConfig(ip)).ToList() ?? null;
+                    }
+                    else if (vmNic.IpConfigs != null)
+                    {
+                        ipConfigList = this.IPConfig.ToList();
+                        var inputIPConfigNames = this.IPConfig.Select(ip => ip.IPConfigName).ToList();
```

Comment:
This is only being used for Contains check. Contains should be present for IEnumerable. No need to call ToList()

Reply:
done